src/help/guides/extension-development/function-definitions.md

```diff
@@ -1,13 +1,18 @@
 # Required Methods
 ## getMangaDetails
+### Method signature
 `async getMangaDetails(mangaId: string): Promise<Manga>`
 
-- A manga ID is defined by the source developer. However IDs are provided to Paperback in other functions, will be given to this function in the same format.
+### Parameters
+| Parameter | Type | Description|
+|-----------|------|------------|
+|  mangaId  | String | The ID of a manga. The manga ID is provided from the other discovery functions, such as the home page and performed searches. The manga ID should be used to return information about the manga, such as the title and author.|
```

Comment:
What would you think of using a code bloc? I'm not sure if its better with or without
```suggestion
|  `mangaId`  | String | The ID of a manga. The manga ID is provided from the other discovery functions, such as the home page and performed searches. The manga ID should be used to return information about the manga, such as the title and author.|
```

Reply:
Should this be done for every parameter? I mean it would help distinguish between uppercase I's and lowercase l's.